ironfish/src/utils/blockchain.ts

```diff
@@ -40,6 +42,23 @@ export function getBlockRange(
   return { start, stop }
 }
 
+export const NATIVE_ASSET_VALUE = {
+  createdTransactionHash: GENESIS_BLOCK_PREVIOUS,
+  identifier: Asset.nativeIdentifier(),
+  metadata: Buffer.from("Native asset of Iron Fish blockchain", 'utf8'),
+  name: Buffer.from("$IRON", 'utf8'),
+  nonce: 0,
```

Comment:
@rohanjadvani What value I should put for nonce?

Reply:
I don't know, I would prefer this type to match Asset or else it has to return Asset | typeof NATIVE_ASSET_VALUE

Reply:
Return `Asset | typeof NATIVE_ASSET_VALUE` for the getAssetById